ftp-ballerina/tests/client_endpoint_test.bal

```diff
@@ -534,7 +534,7 @@ public function testRemoveDirectory() {
     }
     if (response2 is boolean) {
         log:printInfo("Existence of the directory: " + response2.toString());
-        // test:assertEquals(response2, false, msg = "Directory was not removed during `rmdir` operation");
+        test:assertEquals(response2, false, msg = "Directory was not removed during `rmdir` operation");
```

Comment:
This line is commented due to the issue [1].

[1] https://github.com/ballerina-platform/ballerina-standard-library/issues/1448

Reply:
As far as I saw this issue also gets fixed from this. Will add that to the PR comment.